config/base.py

```diff
@@ -11,7 +11,8 @@
 import re
 from django.core.exceptions import ImproperlyConfigured
 from django.db import connection
-
+import sys
```

Comment:
Thank you Yulisa and Jack! Could you please roll back (undo) your changes to the config/base.py file?

Reply:
Could you also please roll back the changes to package-lock.json? Thank you in advance.